docs/strategy.md

```diff
@@ -0,0 +1,9 @@
+# k8gb strategy options
+## failover
+Pinned to a specified primary cluster until workload on that cluster has no available Pods, upon which the next available cluster's Ingress node IPs will be resolved. When Pods are again available on the primary cluster, the primary cluster will once again be the only eligible cluster for which cluster Ingress node IPs will be resolved
+
+## roundRobin (*default*)
```

Comment:
Please place this strategy first on the list. What do you mean by *default* ?

Reply:
@ytsarev default means this is default strategy

Reply:
@k0da yeah. But why is it default? we always specify the strategies explicitly

Reply:
@ytsarev inspired by docs I'm changing:
`* **Round robin** - The default strategy`

Reply:
@k0da still I am challenging it :) It's not real default one :)

Reply:
@ytsarev as you wish, amended